index.html

```diff
@@ -192,6 +192,33 @@ <h3 class="head">🔸Skill</h3>
                         
                     </div>
                 </section>
-               
+           <section class="section_3">
+            <h2 class="description-2">I'm always interested in hearing about new projects,
+                       so if you'd like to chat please get in touch.</h2>
+                       <form action="">
+                      <label for="name-input">
+                        <input type="text" name="name-input" id="name-input" maxlength = "30" placeholder="    Full name" required>
+                      </label><br>
+                      <label for="email-input">
+                        <input type="email" name="email-input" id="email-input" placeholder="    Email address" required>
+                      </label><br>
+                      <label for="more-text">
+                        <textarea name="" id="more-text" cols="30" rows="6" maxlength="500" placeholder="Enter text here"></textarea required>
```

Comment:
- [ ] Good job using `required` attribute for all the elements:+1:, but I've noticed that your text area is not using the `required`. It could be because it is set on the close tag of it. 
Your code:
```
<textarea name="" id="more-text" cols="30" rows="6" maxlength="500" placeholder="Enter text here"></textarea required>
```
What it should be:
```
<textarea name="" id="more-text" cols="30" rows="6" maxlength="500" placeholder="Enter text here" required></textarea>
```